src/mscorlib/shared/System/Diagnostics/Debug.cs

```diff
@@ -13,6 +13,7 @@ namespace System.Diagnostics
     public static partial class Debug
     {
         private static readonly object s_lock = new object();
+        private enum ErrorSource { Assert=1, Assume }
```

Comment:
You should add comments both here and in eepolicy.h indicating that these enumerations mirror one another and must be kept in sync. I would also name this one identical to what is eepolicy.

Reply:
Can we just pass this as a string instead of enum?

Reply:
My initial implementation was passing around string, but I thought it was easier to pass around an enum. I'll fix it to string. Thanks!